napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -524,6 +536,9 @@ def handle_action(self, item, pkg_name, action_name, update=False):
         elif action_name == "cancel":
             widget.set_busy(trans._("cancelling..."), update)
             method((pkg_name,))
+        elif action_name == "help":
+            import webbrowser
+            webbrowser.open("https://napari-hub.org/plugins/" + pkg_name)
```

Comment:
i think i have a slight preference for using the `home_page` from the standard python metadata.  If a plugin choses, they can certainly make their hub page their homepage, but it allows some flexibility and uses a standard python metadata field.  You can access that url at `project_info.url` , and if it's `None` or empty, perhaps fallback to hub then?

Reply:
I'm happy to do that change. I was just thinking this could be a good opportunity to advertise the napari-hub. Your suggestion has the additional benefit that it also works for plugins that are not available on the hub. Let's wait a day for others to comment. If nobody objects and/or has a better idea, I will implement your suggestion :-)

Reply:
>i think i have a slight preference for using the home_page from the standard python metadata.

Me too :)

Reply:
> every plugin will have a listing on the napari hub, unless the developer has explicitly opted out

... or the plugin is shared in non-public ways. A lot of scientists (at least here on campus) share plugins and code in non-public ways before the project is published in a scientific journal..